frontend/static/messages/en.json

```
@@ -1477,6 +1479,11 @@
       "defaultIndicator": "Default input source"
     },
     "main": {
+      "errors": {
```

Biome (2.4.4)

[error] The key errors was already declared.

(lint/suspicious/noDuplicateObjectKeys)